src/freenas/usr/local/sbin/ctdb_glfs_lock

```diff
@@ -168,6 +188,37 @@ def main():
         write_stdout(3)
         sys.exit(1)
 
+    try:
+        with open(RECLOCK_HELPER_PIDFILE, 'r') as f:
+            existing_pid = int(f.read())
+    except (ValueError, FileNotFoundError):
+        existing_pid = PID_UNKNOWN
+
+    if args.kill:
+        if existing_pid == -2:
+            sys.exit(0)
+
+        if existing_pid == -1:
+            raise ValueError('Invalid pid in pidfile!')
+
+        os.kill(existing_pid, signal.SIGTERM)
+        os.remove(RECLOCK_HELPER_PIDFILE)
+        sys.exit(0)
+
+    if existing_pid != PID_UNKNOWN:
+        # optimization to avoid opining glusterfs handle if
```

Comment:
opining is probably supposed to be opening? Haha but stating an opinion on the gluster handle is fine too